packages/lesswrong/components/ea-forum/EABestOfPage.tsx

```diff
@@ -168,7 +169,7 @@ const EABestOfPage = ({ classes }: { classes: ClassesType }) => {
     fragmentName: "PostsListWithVotes",
     terms: {
       view: "curated",
-      after: moment(currentTime).subtract(1, "month").startOf("day").toDate(),
+      curatedAfter: moment(currentTime).subtract(1, "month").startOf("day").toDate(),
```

Comment:
I also fixed this, so that newly-curated posts properly appear in the "Highlights this month" section